doc/tutorials/imgproc/imgtrans/hough_lines/hough_lines.markdown

```diff
@@ -97,6 +97,12 @@ b.  **The Probabilistic Hough Line Transform**
     of the detected lines \f$(x_{0}, y_{0}, x_{1}, y_{1})\f$
 -   In OpenCV it is implemented with the function **HoughLinesP()**
 
+c.  **The Weighted Hough Transform**
```

Comment:
Line 86 should be "OpenCV implements three kind of Hough Line Transforms:"

Reply:
I reviesed it.